overdrive.py

```diff
@@ -530,6 +530,9 @@ def parse_roles(cls, id, rolestring):
 
     @classmethod
     def book_info_to_circulation(cls, book):
+        """ Note:  The json data passed into this method is from a different file 
+        from the json data that goes into the book_info_to_circulation() method.
```

Comment:
this comment is talking about a different method: book_info_to_metadata or something like that